components/template-mgt/org.wso2.carbon.identity.template.mgt/src/test/java/org/wso2/carbon/identity/template/mgt/dao/impl/TemplateManagerDAOImplTest.java

```diff
@@ -47,6 +47,7 @@
 import static org.wso2.carbon.identity.template.mgt.util.TestUtils.mockDataSource;
 import static org.wso2.carbon.identity.template.mgt.util.TestUtils.spyConnection;
 
+@Deprecated
```

Comment:
remove deprecated tag from the classes in org.wso2.carbon.identity.template.mgt module

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2996/commits/1aa997d374f2254338f5287b2bc2fef815104ff3